src/util/brightcoveHelpers.tsx

```diff
@@ -0,0 +1,99 @@
+import { contributorTypes, contributorGroups } from '@ndla/licenses';
```

Comment:
Sett kjapt over bare, men disse util-funksjonene støtter kun bokmål? Og finnes det eksisterende utilfunksjon i ndla-licenses som dekker getContributorGroups?

Reply:
Må tilpasse den for Brightcove videoene siden den sender oss et objekt som kan være av typen `Record<String,any>` som ikke kan types i GQL. Kan forsåvidt eventuelt gjøre om transformasjonen som skjer i graphql for å så ta i bruk licenses metoden. 

Men ser nå at det ikke finnes funksjonalitet som dekker problemstillingen.